readme.md

```diff
@@ -28,15 +28,60 @@ In order to make changes to the database schema a local Supabase instance is nee
 * Committed migrations that are merged to the main branch are automatically deployed.
 
 ### Data
-Until the topdesk api is used as primary data source, we'll read CSV from 
-`data/studieplekken-latest`. When a new version of the data comes in from TU, 
-do the following:
-
-0. convert the incoming file from *xlsx* to *csv*
-0. save to `data` directory as `studieplekken-v<version>.csv`
-0. update the symlink to point to the latest version:
-```sh
-ln -sf studieplekken-v<version>.csv data/studieplekken-latest.csv
+The Spacefinder combines data from different sources:
+
+* **Locations**: the TU Delft provides a list of locations as a CSV file (`data/studieplekken.csv`). The file is generated from a system of TU Delft Education and Student Affairs (ESA) and emailed to the Spacefinder team. Since new CSV files are rarely received, the location data is stored in the codebase and compiled into useable files (`src/data/buildings.json` and `spaces.json`) during build-time. The TU Delft plans to make the location data available through a TopDesk API in the future, but there is no timeline available.
+* **Content**: a DatoCMS instance is used for enriching building content and managing the rich text content used in the application. The use of a CMS makes this content easily editable by (TU Delft members of) the Spacefinder team. The content data is dumped during build-time (`src/data/*page.json`), so there's no run-time dependency on the CMS.
+* **Opening hours**: the TU Delft provides opening hours for (some) buildings and spaces via a Microsoft Exchange web service. ~~The Spacefinder app was originally developed as a static generated website. Opening hours were therefore also added to the locations during build-time. Since, the Spacefinder does not receive updates on changes in opening hours, a cron job is used to trigger a nightly build (`.github/workflows/scheduled-build.yml`).~~ ❌ The Exchange web service is currently unavailable due to expired credentials. The Spacefinder team is waiting for TU Delft support. The opening hours are therefore hidden in the UI (`HIDE_OPENING_HOURS=1`) and the nightly build has been disabled.
```

Comment:
`HIDE_OPENING_HOURS` is part of #152 and it yet to be merged. So double check this is the correct variable name before merging.

Reply:
Will do.

Reply:
Todo: actually remove the [nightly build](https://github.com/voorhoede/tudelft-spacefinder/actions/workflows/scheduled-build.yml)

Reply:
Resolved by https://github.com/voorhoede/tudelft-spacefinder/pull/160/commits/c1ea3a58f2548244ff8a1091306c2c8e0fc4c4f9